mapclassify/greedy.py

```diff
@@ -0,0 +1,347 @@
+"""
+greedy - Greedy (topological) coloring for GeoPandas
+
+Copyright (C) 2019  Martin Fleischmann, 2017 Nyall Dawson
+
+This program is free software: you can redistribute it and/or modify
```

Comment:
@nyalldawson graciously agreed to the license switch.
Can we change the license to [3-clause BSD](https://opensource.org/licenses/BSD-3-Clause)?